src/helix-ui/styles/core/hx-checkbox.less

```diff
@@ -0,0 +1,82 @@
+hx-checkbox {
+  background-color: @gray-0;
+  border-color: currentColor;
+  border-radius: 2px;
+  border-style: solid;
+  border-width: 1px;
+  color: @gray-500;
+  display: inline-block;
+  height: 1rem;
+  vertical-align: middle;
```

Comment:
This was changed from `-2px` to `middle` to fix a style bug when clicking the checkbox.